README.md

```diff
@@ -37,6 +41,8 @@ To get started, please follow these steps:
 
 We welcome contributions to enhance Duckbot! If you find any issues, have suggestions, or want to request a feature, please follow our [Contributing Guidelines](https://github.com/compsci-adl/.github/blob/main/CONTRIBUTING.md).
 
+Before submitting a PR, please make sure your code passes linting checks using `poetry run black`. 
```

Comment:
We really need to get #20 sorted haha. @tinnamchoi Do you want to set this up?

Reply:
still can't fall asleep so ig why not lmao